management-api-agent-dse-6.9/pom.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright DataStax, Inc.
+
+    Please see the included license file for details.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>datastax-mgmtapi</artifactId>
+    <groupId>io.k8ssandra</groupId>
+    <version>${revision}</version>
+  </parent>
+  <version>${revision}</version>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>datastax-mgmtapi-agent-dse-6.9</artifactId>
+  <repositories>
+    <repository>
+      <id>central</id>
+      <url>https://repo.maven.apache.org/maven2</url>
+    </repository>
+    <repository>
+      <id>artifactory-snapshots</id>
+      <url>https://repo.datastax.com/datastax-snapshots-local</url>
+    </repository>
+    <repository>
+      <id>artifactory-releases</id>
+      <url>https://repo.datastax.com/datastax-releases-local</url>
+    </repository>
+  </repositories>
+  <properties>
+    <dse.version>6.8.47</dse.version>
```

Comment:
This will need to change at some point, but for now, let's assume that compiling the DSE 6.9 agent against 6.8 libraries won't break things.

Reply:
probably needs to be updated to 6.8.48 too